<launch>
    
    <node pkg ="my_turtlebot_topics"
        type="rec_odom_action_srv.py"
        name="record_odom_action_server_node"
        output="screen">
  </node>
  
  <node pkg ="my_turtlebot_topics"
        type="obstacle_detect_srv.py"
        name="obstacle_detect_srv_node"
        output="screen">
  </node>

  <node pkg ="my_turtlebot_topics"
        type="main_program.py"
        name="turtlebot_main_node"
        output="screen">
  </node>
    
</launch>